coderd/workspaceapps/appurl/appurl.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 var (
-	// Remove the "starts with" and "ends with" regex components.
-	nameRegex = strings.Trim(UsernameValidRegex.String(), "^$")
+	// nameRegex is the same as our UsernameRegex without the ^ and $.
+	nameRegex = "[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*"
```

Comment:
I think it should still trim the old regex instead TBH

Reply:
Because of import loops, we would need to put that regex in some other package then :cry:.

Reply:
codersdk maybe?